x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/file/FileUserPasswdStoreTests.java

```diff
@@ -51,6 +52,11 @@
     private Environment env;
     private ThreadPool threadPool;
 
+    @BeforeClass
+    public static void disableInFips(){
+        assumeFalse("We are using hashing algorithms that won't be available in FIPS-140 mode", inFipsJvm());
+    }
```

Comment:
I don't think we should disable this whole test in FIPS. It seems like an unnecessary reduction in test coverage.

Reply:
Sure, that was a lazy decision